xbmc/Application.cpp

```diff
@@ -2990,6 +2990,11 @@ bool CApplication::PlayStack(const CFileItem& item, bool bRestart)
   CFileItem selectedStackPart = m_stackHelper.GetCurrentStackPartFileItem();
   selectedStackPart.m_lStartOffset = startoffset;
 
+  std::string savedPlayerState = item.GetProperty("savedplayerstate").asString();
+  if (!savedPlayerState.empty())
+    selectedStackPart.SetProperty("savedplayerstate", savedPlayerState); // pass on to part
+  item.ClearProperty("savedplayerstate");
```

Comment:
We should use `HasProperty("savedplayerstate")` because doing all the stuff. And why do we need to clear the property here as well?

```
if (item.HasProperty("savedplayerstate"))
{
  selectedStackPart.SetProperty("savedplayerstate", item.GetProperty("savedplayerstate").asString());
  ...
}
```

Reply:
I'll change to use HasProperty. The reason to clear is because we don't need it anymore (it's only temporary use until we get here).